tests/tests_repeatedly.yml

```diff
@@ -0,0 +1,90 @@
+---
+- name: Issue simple self-signed certificate
+  hosts: all
+  become: true
+
+  vars:
+    certificate_requests:
+      - name: mycert
+        dns: www.example.com
+        ca: self-sign
+  roles:
+    - linux-system-roles.certificate
+
+- name: Verify certificate
+  hosts: all
+  become: true
+  vars:
+    certificates:
+      - path: /etc/pki/tls/certs/mycert.crt
+        key_path: /etc/pki/tls/private/mycert.key
+        subject:
+          - CN=www.example.com
+        subject_alt_name:
+          - DNS:www.example.com
+  tasks:
+    - name: Verify each certificate
+      include_tasks: tasks/assert_certificate_parameters.yml
+      loop: "{{ certificates }}"
+      loop_control:
+        loop_var: cert
+
+- name: Test with repeatedly by adding same value for parameter
+  hosts: all
+  become: true
+
+  vars:
+    certificate_requests:
+      - name: mycert
+        dns: www.example.com
+        ca: self-sign
+  roles:
+    - linux-system-roles.certificate
```

Comment:
How does this test for idempotency?  That is, how are you verifying that the role didn't change anything?  I guess since `name` is a unique key, you are guaranteed that this did not add a new certificate request for `mycert`, but you have no way to know if Ansible reported `changed=True` for some task in the role. @pcahyna do you know if there is any way to do this?  AFAIK, Ansible does not expose the internal statistics gathering except upon the final output of the `ansible-playbook` command as `changed=N` where `N` is an integer.  As in this proposal to implement support for idempotency checking - https://github.com/linux-system-roles/timesync/pull/72/files#diff-cfddcd75655d3ce8647e77f40dd7baf9R48-R71 - which looks for `CHANGED=>` in the output